packages/documentation-site/patternfly-docs/content/get-started/release-notes-data.js

```diff
@@ -0,0 +1,4854 @@
+    pullRequestURL: "https://github.com/patternfly/patternfly/pull/5422 ",
+    details: "Removed unnecessary line-height styles from the options menu toggle.",
+    repo: "HTML/CSS",
+    fixedWithCodeMod: false
+  },
+  {
+    component: "Page",
+    description: "Added multiple body support",
+    pullRequestURL: "https://github.com/patternfly/patternfly/pull/5093",
+    details: "Updated the page component sidebar to support multiple body elements, instead of a single body element used to position vertical navigation.",
+    repo: "HTML/CSS",
+    fixedWithCodeMod: false
+  },
+  {
+    component: "Page",
+    description: "Default page to full viewport height",
+    pullRequestURL: "https://github.com/patternfly/patternfly/pull/5517",
+    details: "Updated the page component height to be 100% of the viewport height using `100vh` and `100dvh` units. Previously the page height was set to 100%, requiring the need for all ancestors of the page component element to also have an implicit or explicit CSS height set. This update removed the need for any ancestors of the page component to have a height set.",
+    repo: "HTML/CSS",
+    fixedWithCodeMod: false
+  },
+  {
+    component: "Page",
+    description: "Moved variables to root",
+    pullRequestURL: " https://github.com/patternfly/patternfly/pull/5637",
+    details: "This update scopes all of the page component CSS variables to the `:root` scope instead of the page component class selector. This allows any top-level CSS variable for the page component to be accessible from anywhere in the document, instead of only being accessible within the page component class selector.",
+    repo: "HTML/CSS",
+    fixedWithCodeMod: false
+  },
+  {
+    component: "Pagination",
+    description: "Updated styles to target menu toggle",
+    pullRequestURL: "https://github.com/patternfly/patternfly/pull/5619 ",
+    details: "Updated all styles within the pagination component that were targeting the deprecated options menu toggle to target the menu toggle component instead.",
+    repo: "HTML/CSS",
+    fixedWithCodeMod: false
+  },
+  {
+    component: "Popover",
+    description: "Added header, title, close elements",
+    pullRequestURL: "https://github.com/patternfly/patternfly/pull/5370 ",
+    details: "Refactored the popover component HTML to be more flexible and consistent with similarly structured components. Added a structural HTML element to the container that is used to wrap the close button. Made the header element required. Updated the popover title to be more flexible",
+    repo: "HTML/CSS",
+    fixedWithCodeMod: false
+  },
+  {
+    component: "Progress",
+    description: "Prevented empty description and status from rendering",
+    pullRequestURL: "https://github.com/patternfly/patternfly/pull/5387",
+    details: "The title isn’t a breaking change. However, this pull request also updated the progress status element to use CSS `gap` to space its children instead of `margin`.",
```

Comment:
Oh when I wrote "The title isn’t a breaking change. However...", I didn't intend on it being in the release notes. It's more my commentary 😂 I might update this to:

```suggestion
    description: "Use gap instead of margin for status box",
    pullRequestURL: "https://github.com/patternfly/patternfly/pull/5387",
    details: "Updates the progress status element to use CSS `gap` to space its children instead of `margin`, which impacts the space between the status icon text specifically.",
```